internal/drivers/azure/driver.go

```diff
@@ -211,6 +221,13 @@ func (c *config) Create(ctx context.Context, opts *types.InstanceCreateOpts) (in
 		},
 	}
 
+	if c.id != "" {
```

Comment:
This should check for security type right?
```if c.securityType != ""```

Reply:
We are taking ID as primary